test_runner/src/main/kotlin/ftl/util/Utils.kt

```diff
@@ -169,8 +169,7 @@ fun withGlobalExceptionHandling(block: () -> Int) {
 }
 
 private fun SavedMatrix.logError() {
-    println("More details are available at [${this.webLink}]")
-    println(this.asPrintableTable())
+    println("Matrix is $state")
```

Comment:
There will be no table output if flank throw FTLError before MatrixResultsReport?

Reply:
`FTLError` is throw after printing table
```

listOf(
    CostReport,
    MatrixResultsReport
).map {
    it.run(matrices, testSuite, printToStdout = true, args = args)  >>>>>>>>>>>>> PRINTING TABLE
}
if (!matrices.allSuccessful()) {
    listOf(
        HtmlErrorReport
    ).map { it.run(matrices, testSuite, printToStdout = false, args = args) }
}
JUnitReport.run(matrices, testSuite?.apply {
    if (ignoredTestCases.isNotEmpty()) {
        testsuites?.add(ignoredTestCases.toJunitTestsResults())
    }
}, printToStdout = false, args = args)
when {
    args.fullJUnitResult -> processFullJunitResult(args, matrices, testShardChunks)
    args.useLegacyJUnitResult -> processJunitXml(testSuite, args, testShardChunks)
}
matrices.validateMatrices(args.ignoreFailedTests) >>>>>>>>>>> FTLError
```